apps/src/templates/Notification.jsx

```diff
@@ -64,7 +64,7 @@ const styles = {
     float: 'left',
   },
   contentBox: {
-    width: 'calc(100% - 72px)',
+    width: 'calc(100% - 100px)',
```

Comment:
This number is starting to feel magic; a quick comment about where the 100px comes from?